src/applications/check-in/travel-claim/README.md

```diff
@@ -0,0 +1,135 @@
+# Travel Claim Application
+
+## Description
+
+The Travel Claim Application satifies a travel only flow. There is no check in or status changes to the appointments. It uses shared components from the other apps(pre-check-in and check-in). It is designed to allow patients at Oracle Health centers to file a travel claim on the day of their appointment or appointments.
+
+## Slack Channels
+
+- [Check In Experience](https://slack.com/app_redirect?channel=C022AC2STBM)
+
+## Approval Groups
+
+- [Health Care Experience Team](https://github.com/orgs/department-of-veterans-affairs/teams/vsa-healthcare-experience)
+- [Health Care System Team](https://github.com/orgs/department-of-veterans-affairs/teams/va-cto-health-products)
+
+## Project Documentation
+
+## Good to knows
+
+### Project URL
+
+``` markdown
+/my-health/appointment-travel-claim/
+```
+
+### What version of the api are we using?
+
+What we know
+
+- Using v2 of the api for low auth
+- GET from vets-api for appointment and patient data
+- POSTing to a vets-api BTSSS endpoint to file a travel claim
+
+### How to run locally
+
+Follow the standard directions to run the app. The API needs to be running in order to run the app locally. Currently I would use the mock api in `src/applications/pre-check-in/api/local-mock-api` using the directions in the [README](https://github.com/department-of-veterans-affairs/vets-website/blob/main/README.md#running-a-mock-api-for-local-development). This makes development easier since creating a valid token is tedious.
```

Comment:
Not a change that you need to make but I would suggest saying TBD on this until we build out the mocks